### PR TITLE
Remove tabzilla, add mofo-footer

### PR DESCRIPTION
**Pull request by alanmoo**

### CODE REVIEW

File: components/footer.jsx [hunk 1]
```diff
@@ -2,6 +2,42 @@ var React = require('react');
...
+const ORGS = [
+  {
+    name: `mozilla`,
+    link: `https://mozilla.org`,
+    description: (<p>Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the web for the public good. Visit us at <a href="https://mozilla.org">mozilla.org</a>.</p>),
+    className: `mozilla`
+  }
+];
+
+export { Footer as default };
```
Comment (mmmavis) [inline components/footer.jsx]: we can remove this, right?

File: components/footer.jsx [hunk 2]
```diff
@@ -2,6 +2,42 @@ var React = require('react');
 var Router = require('react-router');
```
Comment (mmmavis) [inline components/footer.jsx]: since you are on this file already, can we change these to `import` to make things look more unified?

**Comment by mmmavis** [comment]
`yarn.lock` probably doesn't have to be committed?

**Comment by alanmoo** [comment]
Leaving yarn.lock because it does help consistency in installs if multiple people are using yarn. Heroku also uses it and respects the engine so we get more consistent deploys.

**Pull request merged**